hcl2template/parser.go

```diff
@@ -60,6 +62,8 @@ type Parser struct {
 	ProvisionersSchemas packer.ProvisionerStore
 
 	PostProcessorsSchemas packer.PostProcessorStore
+
+	DataSourceSchemas packer.DataSourceStore
```

Comment:
👍🏼

Reply:
Well said let me see if I can get any more info on the second pass